objgraph.py

```diff
@@ -718,25 +734,28 @@ def show_graph(objs, edge_func, swap_source_target,
         dot.wait()
         if dot.returncode != 0:
             # XXX: shouldn't this go to stderr or a log?
-            print("dot failed to generate '%s' image: output format not supported?")
+            print('dot failed to generate "%s" image:',
+                  'output format not supported?')
```

Comment:
Speaking of bugs I JUST NOTICED THERE'S AN UNEXPANDED %s IN THERE!  AUGH I hate when other programs to that to me :)